tests/e2e/util.go

```diff
@@ -804,7 +804,10 @@ func setResourceQuota(client clientset.Interface, namespace string, size string,
 	defer cancel()
 	existingResourceQuota, err := client.CoreV1().ResourceQuotas(namespace).Get(ctx, namespace, metav1.GetOptions{})
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
-
+	framework.Logf("existingResourceQuota name %s", existingResourceQuota.GetName())
+	requestStorageQuota := existingTestResourceQuota(existingResourceQuota.GetName(), size, namespace)
+	testResourceQuota, err := client.CoreV1().ResourceQuotas(namespace).Update(ctx, requestStorageQuota, metav1.UpdateOptions{})
+	ginkgo.By(fmt.Sprintf("ResourceQuota details: %+v", testResourceQuota))
 	newQuota := newTestResourceQuota(existingResourceQuota.GetName(), size, scName)
 	resourceQuota, err := client.CoreV1().ResourceQuotas(namespace).Update(ctx, newQuota, metav1.UpdateOptions{})
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Do we have to wait for the quota to show up in the namespace describe output after this step? I remember there was a bug related to this.

Reply:
Done